### PR TITLE
GET /access/policies

### PR DESCRIPTION
**Pull request by jonmattgray**
Add an access controller with a single `GET /access/policies` route to get all access policies.
Uses Open Policy Agent to handle policies
Includes a sample policy (`.rego`) which describes that a customer can only query a supplier if the customer is in the supplier's list of customers

### CODE REVIEW

File: docker-compose-agent-ipfs.yml [hunk 1]
```diff
@@ -55,6 +55,21 @@ services:
...
 
+  ##################################################################################
+  ## OPA ###########################################################################
+  ##################################################################################
+
+  opa:
+    container_name: opa
+    image: openpolicyagent/opa:0.60.0-static
+    ports:
+      - '8181:8181'
```
Comment (rmlearney-digicatapult) [inline docker-compose-agent-ipfs.yml:66]: Do we want this exposed outside the container?

File: docker-compose-testnet.yml [hunk 2]
```diff
@@ -190,6 +190,21 @@ services:
...
 
+  ##################################################################################
+  ## OPA ###########################################################################
+  ##################################################################################
+
+  opa:
+    container_name: opa
+    image: openpolicyagent/opa:0.60.0-static
+    ports:
+      - '8181:8181'
```
Comment (rmlearney-digicatapult) [inline docker-compose-testnet.yml:201]: Same - do we want this exposed outside the container?

**Review by Ellenn-A — APPROVED** [review]
seems legit

File: docker-compose-agent-ipfs.yml (hunk 1 above)
Comment (jonmattgray) [reply docker-compose-agent-ipfs.yml:66]: I guess not in the real world. It's useful for developing against to be able to send HTTP requests directly to OPA

**Pull request merged**